index.html

```diff
@@ -275,6 +275,7 @@ <h2 data-i18n="dgc-getting-started" class="page-section-heading text-center text
                           <li><a class="dropdown-item select-lang" data-lang="id">Bahasa Indonesia</a></li>
                           <li><a class="dropdown-item select-lang" data-lang="mr">Marathi</a></li>
                           <li><a class="dropdown-item select-lang" data-lang="tr">Türkçe</a></li>
+                          <li><a class="dropdown-item select-lang" data-lang="cn">简体中文</a></li>
```

Comment:
should be zh-cn

Reply:
Don't forget to add "Chinese" option into guide.html and support.html